assets/messages.yml

```diff
@@ -441,7 +441,7 @@ messages:
       fillname: []
   duplicate-of-mcl-14369:
     - project: mcl
-      name: Duplicate of MCL-14369
+      name: Duplicate of MCL-14369 (All versions missing in launcher)
```

Comment:
This might not be completely accurate because the bug that all versions are missing only seems to be one variant of this. However, this text here is only supposed to be a hint what the report is about. (Additionally that bug might not actually occur anymore?)

So I am leaving it like this for now.